src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -694,7 +691,6 @@
                        :params     [{:uuid       (get-in transaction-for-signing [:sendDetails :uuid])
                                      :signatures signatures-map}]
                        :on-success (fn []
-                                     (rf/dispatch [:wallet/clean-send-data])
```

Comment:
`:wallet/clean-send-data` was moved to `:wallet/clean-up-transaction-flow`, so we don't clean up the state before the components were unmounted.